plugins/modules/accesspoint_workflow_manager.py

```diff
@@ -1160,57 +1230,65 @@ def verify_diff_merged(self, config):
         require_update = self.config_diff(self.have["current_ap_config"])
         self.log(self.pprint(require_update), "INFO")
         if require_update:
-            radio_list = require_update.get("radioConfigurations")
+            radio_list = require_update.get("radioConfigurations", [])
             if len(radio_list) > 0:
                 for each_radio in radio_list:
                     radio_key_list = list(each_radio.keys())
                     for each_key in radio_key_list:
-                        if each_key not in ("antenna_name", "radioType", "unmatch", "cable_loss", "radioRoleAssignment"):
+                        if each_key not in ("antenna_name", "radioType", "unmatch", "cable_loss",
+                                            "radioRoleAssignment"):
```

Comment:
why do we use radioRoleAssignment in camel case?

Reply:
Maped with keymap